app/views/dashboard/index.html.erb

```diff
@@ -26,25 +26,25 @@
   <div class="govuk-grid-column-one-third">
     <div class="wrapper">
       <h1 class="govuk-heading-s">
-        <%= link_to "See all allocated prisoners", allocated_prison_prisoners_path(@prison.code), class:"govuk-link" %>
+        <%= link_to "Make new allocations", unallocated_prison_prisoners_path(@prison.code), class:"govuk-link" %>
       </h1>
-      <p class="govuk-body">All prisoners who have been allocated to a Prison Offender Manager.</p>
+      <p class="govuk-body">You need to allocate <%= @caseloads.count %> cases to Prison Offender Manager(POM)</p>
     </div>
   </div>
   <div class="govuk-grid-column-one-third">
     <div class="wrapper">
       <h1 class="govuk-heading-s">
-        <%= link_to "Make new allocations", unallocated_prison_prisoners_path(@prison.code), class:"govuk-link" %>
+        <%= link_to "Update case information", missing_information_prison_prisoners_path(@prison.code), class:"govuk-link" %>
       </h1>
-      <p class="govuk-body">Prisoners who need to be allocated to a Prison Offender Manager.</p>
+      <p class="govuk-body">You need to add missing detail to <%= @caseloads.count %> cases so that they can be allocated to a POM .</p>
```

Comment:
I think we need a test here. You're using the same count for both these sections which doesn't look right, and I'm really not sure what is in the variable 'caseloads' - seems a strange name